go.mod

```diff
@@ -310,3 +310,6 @@ replace gopkg.in/yaml.v2 => github.com/juju/yaml/v2 v2.0.0
 // methods, which sometimes clash with package names. Fix the package to v0.4.0
 // until the issue is resolved.
 replace go.uber.org/mock => go.uber.org/mock v0.4.0
+
+// TODO - the tip of names v6 has a space ID parsing bug
```

Comment:
This should be a separate commit at the least. The change is sneaking in without comment as is.

Reply:
This was added because go mod tidy updates to the tip of names/v6 which brings in the spaces bug. The line here actually avoids bringing in the bad upstream change. I am not sure how previous PRs which landed got away without needing this line.